src/components/auth/Welcome.js

```diff
@@ -0,0 +1,175 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { View } from 'react-native'
+
+// import logger from '../../lib/logger/js-logger'
+import { fireEvent, INVITEWELCOME_NEXT, INVITEWELCOME_SKIPPED } from '../../lib/analytics/analytics'
+import { withStyles } from '../../lib/styles'
+import { getShadowStyles } from '../../lib/utils/getStyles'
+import AsyncStorage from '../../lib/utils/asyncStorage'
+import { IS_FIRST_VISIT } from '../../lib/constants/localStorage'
+import CustomButton from '../common/buttons/CustomButton'
+import Wrapper from '../common/layout/Wrapper'
+import Text from '../common/view/Text'
+
+import {
+  getDesignRelativeHeight,
+  getDesignRelativeWidth,
+  isShortDevice,
+  isVeryShortDevice,
+} from '../../lib/utils/sizes'
+import { isBrowser } from '../../lib/utils/platform'
+import AuthNavBar from '../appNavigation/AuthNavBar'
+import AuthProgressBar from '../appNavigation/AuthProgressBar'
+import Illustration from '../../assets/Auth/torusIllustration.svg'
+
+// const log = logger.child({ from: 'Welcome' })
+
+const InviteWelcome = ({ styles, screenProps, navigation }) => {
+  const [step, setStep] = useState(1)
+  const [show, setShow] = useState(undefined)
```

Comment:
All the `InviteWelcome` Logic should be removed, as `InviteWelcome` stays as it is, and this screen will replace the SignUP

Reply:
@TomasGonzalez this is a new welcome screen (actually 'get started'
`components/auth/Welcome.js`.
let's rename it to GetStarted to avoid confusing

the old welcome screens are untouched, they located in `src/components/inviteWelcome`